R/convert.R

```diff
@@ -126,6 +149,12 @@ convert_gene <- function(df, frm, to, species = "human", frm_cols = NULL, quiet
     df_out[, col] <- new_genes[[col]]
   }
   
+  # Remove row numbers column
+  df_out <- subset(df_out, select = -c(id))
+
+  # Replace NoData with NA
+  df_out[df_out == "NoData"] <- NA_character_
+
   return(df_out)
```

Comment:
you don't need to explicitly use `return` as you do in python, but you can if you want to.

Reply:
I didn't know that, so thank you for telling me! I think I'll keep it for readability for myself for now.